applications/convection_diffusion_application/license.txt

```diff
@@ -0,0 +1,20 @@
+Kratos Convection Diffusion Application
+
+Copyright (c) 2018, Riccardo Rossi, Jordi Cotela Dalmau, CIMNE (International Center for Numerical Methods in Engineering)
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+	-	Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+	-	Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer 
+		in the documentation and/or other materials provided with the distribution.
+	-	All advertising materials mentioning features or use of this software must display the following acknowledgement: 
+			This product includes Kratos Structural Mechanics technology.
```

Comment:
Convection Diffusion Technology ;-)

Reply:
Any better suggestion?

Reply:
Do we need a separate license for the convection diffusion application? I'm partial to releasing as much as possible with the core Kratos license.

Reply:
as Jordi is suggesting, let's use the general one for this (that is, let's just remove this file)